Mailbox.Tests/DataLoaderTests.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace Mailbox.Tests
+{
+    [TestClass]
+    public class DataLoaderTests
+    {
+        [TestMethod]
+        public void SaveToStream_LoadFromStream_CompareLists()
+        {
+            DataLoader dl = new DataLoader(File.Open("Mailboxes.json", FileMode.OpenOrCreate, FileAccess.ReadWrite));
```

Comment:
DataLoader is disposable and should be cleaned up.
```suggestion
            using DataLoader dl = new DataLoader(File.Open("Mailboxes.json", FileMode.OpenOrCreate, FileAccess.ReadWrite));
```